apps/explorer/src/components/footer/Footer.tsx

```diff
@@ -1,61 +1,131 @@
+                    <SocialLinkedin24 />
+                </Link>
+            </li>
+        </ul>
     );
 }
 
 function Footer() {
     return (
-        <footer className="bg-gray-75 px-5 py-10 md:px-10 md:py-14">
-            <nav className="mx-auto grid grid-cols-1 gap-8 md:mx-0 md:grid-cols-4 md:gap-10 xl:w-1/2">
-                <div className="order-last mx-auto md:order-first md:mt-0">
-                    <div className="h-full space-y-2 md:flex md:flex-col md:justify-between">
-                        <SuiLogoIcon className="mx-auto md:mx-0" />
-                        <div className="mt-auto">
-                            <Text
-                                color="white"
-                                variant="pSubtitleSmall/semibold"
-                            >
-                                &copy;
-                                {`${new Date().getFullYear()} Sui. All
-                                rights reserved.`}
-                            </Text>
-                        </div>
+        <footer className="bg-gray-40 px-5 py-10 md:px-10 md:py-14">
+            <nav className="flex flex-col gap-7.5">
+                <div className="flex w-full flex-col items-center justify-between gap-7.5 md:flex-row">
+                    <div className="h-10 text-hero-dark">
+                        <SuiWordmark />
                     </div>
+                    <FooterLinks />
                 </div>
-                <div className="col-span-1 grid grid-cols-4 md:col-span-3">
-                    <FooterLinks links={footerLinks} />
+
+                <div className="h-[1px] w-full bg-gray-45" />
+                <div className="flex w-full items-center justify-between">
+                    <div className="h-full space-y-2">
+                        <Text
+                            color="steel-darker"
+                            variant="pSubtitleSmall/medium"
+                        >
+                            &copy;
+                            {`${new Date().getFullYear()} Sui. All
+                                rights reserved.`}
+                        </Text>
+                    </div>
+                    <ul className="flex  gap-2">
+                        <li>
+                            <Link
+                                variant="text"
+                                href="https://mystenlabs.com/#community"
+                            >
+                                <Text
+                                    variant="pSubtitleSmall/medium"
+                                    color="steel-darker"
+                                >
+                                    Terms & Conditions
+                                </Text>
+                            </Link>
+                        </li>
+                        <li>
+                            <Link
+                                variant="text"
+                                href="https://mystenlabs.com/#community"
```

Comment:
wrong link 🥸

Reply:
whooops, forgot to push latest. sry for the premature request